src/modelmanager.cpp

```diff
@@ -150,6 +150,19 @@ void processNodeInputs(const std::string nodeName, const rapidjson::Value::Const
     }
 }
 
+void processPipelineInputs(const rapidjson::Value::ConstMemberIterator& pipelineInputsPtr, const std::string& nodeName, std::unordered_map<std::string, std::string>& nodeOutputNameAlias, const std::string& pipelineName) {
+    for (const auto& pipelineInput : pipelineInputsPtr->value.GetArray()) {
+        const std::string pipelineInputName = pipelineInput.GetString();
+        SPDLOG_INFO("Alliasing node:{} output:{}, under alias:{}",
+            nodeName, pipelineInputName, pipelineInputName);
+        if (nodeOutputNameAlias.count(pipelineInputName) > 0) {
+            SPDLOG_WARN("Pipeline {} has duplicated input declaration", pipelineName);
```

Comment:
there is not warning level. Should it be error?

Reply:
This is related to bug I created: https://jira.devtools.intel.com/browse/CVS-43541  
For now I just added warning if inputs are duplicated. Such pipeline works fine. Ideal solution would be to restrict administrator from creating such pipelines. This is expained in the jira bug.

Reply:
Done